docs/docsite/rst/network/getting_started/first_playbook.rst

```diff
@@ -150,4 +150,45 @@ The extended first playbook has four tasks in a single play. Run it with the sam
    vyos.example.net           : ok=6    changed=1    unreachable=0    failed=0
 
 
-This playbook is useful. However, running it still requires several command-line flags. Also, running a playbook against a single device is not a huge efficiency gain over making the same change manually. The next step to harnessing the full power of Ansible is to use an inventory file to organize your managed nodes into groups with information like the ``ansible_network_os`` and the SSH user.
+
+.. _network_gather_facts:
+
+Gathering facts from network devices
+====================================
+
+Ansible 2.9 introduced improved fact gathering for network devices. The network ``*_facts`` modules (such as :ref:`eos_facts <eos_facts_module>`) now gather facts and return them in standardized key/value pairs that you can feed into further tasks to manage the network device. You can also use the new ``gather_network_resources`` parameter in these network ``*_facts`` modules to return just a subset of the device configuration.
```

Comment:
You also get this automatically via gather_facts: and should not need to invoke these directly anymore.